### DISCUSSION TITLE
G6 brand new sensor: "sensor failed start"

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#2259] [Q&A]
Today I started a new G6 sensor; the code was accepted as usual, but during the warm up (more precisely, after 35 minutes), I got a "sensor failed start" error message!
I started the sensor again and at the moment I'm still in the warm up phase,with no errors so far.
No sure about what to do now: maybe the sensor is defective?
By the way, I realized that the the trasmitter is 84 days old, so it's going to expire in the next few days.
I'm a bit worried, since on saturday I'll be away for a two-weeks holiday...

**Comment by jonprogers** [comment]
Your tx should be fine restarting at 84 days, up to 99 days will work - see https://navid200.github.io/xDrip/docs/Transmitter-lifetime.html

I'd wait to see what happens during your current warm up cycle. If it fails to start again I'd suspect a bad sensor so dexcom support will advise and probably replace it for you.

Good luck...

**Comment by Navid200** [comment]
As that hyperlink states, you can continue to start sensors as long as transmitter days is 99 or less.
So, when your current sensor ends, it will show 94 and you will still be able to start another sensor.
If you start that one then, it will end when it shows 104 days.  Then, you will not be able to start another.

Sensor start could fail if you don't have proper connectivity:  https://navid200.github.io/xDrip/docs/Proper-connectivity.html
Do you?

**Comment by dagord** [comment]
Hello
just as an update: the warm-up finished without errors and the sensor is now working correctly.
Not sure what happened: I started it as usual and, yes, having proper connectivity (I keep the smartphone in my pocket).
Since the sensor is activated, do you think that I can run into related issues during normal usage? (just wondering)

By the way, did you experience problems in getting support from Dexcom, as unofficial application users?

**Reply by Navid200** [reply]
These are the recommended settings:  https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html
Those include settings on the phone, not just xDrip to ensure that xDrip can function properly.
There are also phones that re known to have issues.

If the reason for issues is that the settings are wrong or the phone keeps dropping connection, it will not be fair to contact Dexcom asking for support.

**Reply by dagord** [reply]
I'm not blaming xDrip... I'm wondering if I can get support for a defective sensor even if I'm not using the official apps.
Maybe a sensor not reading correctly the values or something similar (just saying random things)
I already called Dexcom phone support in the past since after placing a sensor I had blood loss; as a first question, they asked if I was using a smartphone or the standalone reader (which I was actually using at the time). In that case I had no problems in getting a replacement. But what if I declare that I use an unsupported instrument?

**Reply by jonprogers** [reply]
I'm also a tandem user so if I need replacements I just talk about using G6 with that (or animas pump with G4 prior to that). I've never had an issue with UK support but accept that I focus on certain parts of my setup when talking to them! If you have a reader maybe talk about that...
